app/assets/stylesheets/navbar.css

```diff
@@ -0,0 +1,39 @@
+.navbar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #333; /* Change to your desired background color */
+  padding: 10px 0;
+}
+
+.nav-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  text-align: center;
+}
+
+.nav-list li {
+  display: inline;
+  margin-right: 20px;
+}
+
+.nav-list li:last-child {
+  margin-right: 0;
+}
+
+.nav-list a {
+  text-decoration: none;
+  color: #fff; /* Change to your desired text color */
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.nav-list a:hover {
+  color: #ff5733; /* Change to your desired hover color */
+}
+
+.content {
+  margin: 20px; /* Add space between the navbar and content */
+}
```

Comment:
- [ ] [OPTIOANL] You have done a great piece of work here removing these unnecessary comments might make your work appear more clean and readable. :wink: 
- [ ] [OPTIONAL] Another suggestion will be to improve the accessibility and user experience on your `Navigation` due to the choice of color i.e. black on black; this makes it very difficult for users to see the content clearly, especially for differently abled users with a certain color or visual impairments

![image](https://github.com/leloufadel/Recip-App/assets/99715640/b411fb20-32d8-4b11-9943-f2b35c4803cb)